Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -81,3 +89,60 @@ parameters:
   pool: replicapool
   ...
 ```
+
+### Mirroring
+
+First, enable mirroring for the parent CephBlockPool.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph
+spec:
+  replicated:
+    size: 3
+  mirroring:
+    enabled: true
+    mode: image
+    # schedule(s) of snapshot
+    snapshotSchedules:
+      - interval: 24h # daily snapshots
+        startTime: 14:00:00-05:00
+```
+
+Second, configure the rados namespace CRD with the mirroring enabled:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: namespace-a
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the CephBlockPool CR where the namespace is created.
+  blockPoolName: replicapool
+  mirroring:
+    enabled: true
+    mode: image
+    snapshotSchedules:
+```
+
+Another example explaining, if user wants to mirror namespace on some other remote rados namespace on the peer cluster instead of the same rados namespace, user can specify the name using `remoteNamespace`:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: namespace-a
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the CephBlockPool CR where the namespace is created.
+  blockPoolName: replicapool
+  mirroring:
+    enabled: true
+    mode: image
+    remoteNamespace: "test" # rados namespace in the peer cluster, where namespace-a should be mirrorerd
```

Comment:
If the only difference is the `remoteNamespace` in this example, I'd suggest not having the separate example.

Reply:
Okay